war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2675,7 +2675,7 @@ window.addEventListener("load", function () {
   document.querySelectorAll(".jenkins-form-item").forEach(function (element) {
     if (
       element.querySelector(
-        ".optionalBlock-container > .row-group-start input[type='checkbox'], .optional-block-start input[type='checkbox'], div > .jenkins-checkbox",
```

Comment:
Any ideas how this can be checked for unintended side effects?

Given the minimal benefits to removing these classes, any regression doesn't seem worth it.